### PR TITLE
ICU-20795 update APIChangeReport for ICU4C 65.1

### PR DESCRIPTION
**Pull request by srl295**
[ICU-20795]

- API promotions are in a subsequent commit (#808)
- new Markdown format added
- See https://github.com/unicode-org/icu/pull/807 for the tool change which built this report.

[ICU-20795]: https://unicode-org.atlassian.net/browse/ICU-20795

### CODE REVIEW

File: icu4c/APIChangeReport.md [hunk 1]
```diff
@@ -0,0 +1,870 @@
+
+
+<!--
+ Copyright (C) 2016 and later: Unicode, Inc. and others.
```
Comment (markusicu) [inline icu4c/APIChangeReport.md]: should be 2019 and the real copyright symbol

File: icu4c/APIChangeReport.md [hunk 2]
```diff
@@ -0,0 +1,870 @@
...
+| measunit.h |  <tt>static</tt> MeasureUnit* icu::MeasureUnit::createYearPerson(UErrorCode&amp;) | DraftICU 64 | 
+| nounit.h |  UClassID icu::NoUnit::getDynamicClassID() const | DraftICU 60 | 
+| nounit.h |  icu::NoUnit::NoUnit(const NoUnit&amp;) | DraftICU 60 | 
+| nounit.h |  icu::NoUnit::~NoUnit() | DraftICU 60 | 
+| nounit.h |  <tt>static</tt> NoUnit icu::NoUnit::base() | DraftICU 60 | 
+| nounit.h |  <tt>static</tt> NoUnit icu::NoUnit::percent() | DraftICU 60 | 
+| nounit.h |  <tt>static</tt> NoUnit icu::NoUnit::permille() | DraftICU 60 | 
+| nounit.h |  <tt>static</tt> UClassID icu::NoUnit::getStaticClassID() | DraftICU 60 | 
+| nounit.h |  void* icu::NoUnit::clone() const | DraftICU 60 | 
+| numberformatter.h |  Appendable&amp; icu::number::FormattedNumber::appendTo(Appendable&amp;, UErrorCode&amp;) const | DraftICU 62 |
```
Comment (sffc) [inline icu4c/APIChangeReport.md]: Why are these ones (numberformatter.h and numberrangeformatter.h) still showing draft?

File: icu4c/APIChangeReport.md [hunk 3]
```diff
@@ -0,0 +1,870 @@
...
+| decimfmt.h | const number::LocalizedNumberFormatter&amp; icu::DecimalFormat::toNumberFormatter() const |  DeprecatedICU 64 | (missing)
+| edits.h | UBool icu::Edits::copyErrorTo(UErrorCode&amp;) |  StableICU 59 | (missing)
+| platform.h | <tt>#define</tt> __has_attribute |  Internal | (missing)
+| platform.h | <tt>#define</tt> __has_builtin |  Internal | (missing)
+| platform.h | <tt>#define</tt> __has_cpp_attribute |  Internal | (missing)
+| platform.h | <tt>#define</tt> __has_declspec_attribute |  Internal | (missing)
+| platform.h | <tt>#define</tt> __has_extension |  Internal | (missing)
+| platform.h | <tt>#define</tt> __has_feature |  Internal | (missing)
+| platform.h | <tt>#define</tt> __has_warning |  Internal | (missing)
+| uversion.h | <tt>#define</tt> U_NAMESPACE_BEGIN |  StableICU 2.4 | (missing)
```
[the remark this answers is not in the document]
Comment (srl295) [reply icu4c/APIChangeReport.md]: after RC

File: icu4c/APIChangeReport.md [hunk 4]
```diff
@@ -0,0 +1,870 @@
...
+| basictz.h | void* icu::BasicTimeZone::clone() const |  (missing) | StableICU 3.8
+| datefmt.h | void* icu::DateFormat::clone() const |  (missing) | StableICU 2.0
+| edits.h | UBool icu::Edits::copyErrorTo(UErrorCode&amp;) const |  (missing) | StableICU 59
+| measunit.h | <tt>static</tt> MeasureUnit* icu::MeasureUnit::createAtmosphere(UErrorCode&amp;) |  Draft→StableICU 63
+| measunit.h | <tt>static</tt> MeasureUnit* icu::MeasureUnit::createPercent(UErrorCode&amp;) |  Draft→StableICU 63
+| measunit.h | <tt>static</tt> MeasureUnit* icu::MeasureUnit::createPermille(UErrorCode&amp;) |  Draft→StableICU 63
+| measunit.h | <tt>static</tt> MeasureUnit* icu::MeasureUnit::createPetabyte(UErrorCode&amp;) |  Draft→StableICU 63
+| numfmt.h | void* icu::NumberFormat::clone() const |  (missing) | StableICU 2.0
+| unifilt.h | void* icu::UnicodeFilter::clone() const |  (missing) | StableICU 2.4
+| utrace.h | <tt>enum</tt> UTraceFunctionNumber::UTRACE_UDATA_BUNDLE |  (missing) | StableICU 2.8
```
[the remark this answers is not in the document]
Comment (srl295) [reply icu4c/APIChangeReport.md]: fixed in #808

File: icu4c/APIChangeReport.md [hunk 5]
```diff
@@ -0,0 +1,870 @@
...
+| utrace.h | <tt>enum</tt> UTraceFunctionNumber::UTRACE_UDATA_RESOURCE |  (missing) | StableICU 2.8
```
[the remark this answers is not in the document]
Comment (srl295) [reply icu4c/APIChangeReport.md]: fixed in #808

File: icu4c/APIChangeReport.md (hunk 1 above)
Comment (srl295) [reply icu4c/APIChangeReport.md]: fixed in #807 , will regen

File: icu4c/APIChangeReport.md (hunk 2 above)
Comment (srl295) [reply icu4c/APIChangeReport.md]: see #808

**Comment by srl295** [comment]
Well, perhaps this PR wasn't helpful. Closing in favor of #808

**Pull request closed without merging**